packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -195,6 +195,11 @@ export interface BrowserTracingOptions {
    * Default: (url: string) => true
    */
   shouldCreateSpanForRequest?(this: void, url: string): boolean;
+
+  /**
+   * Is called when spans are started for outgoing requests.
+   */
+  onRequestSpanStart(span: Span, requestInformation: { headers?: WebFetchHeaders }): void;
```

Comment:
```suggestion
  onRequestSpanStart?(span: Span, requestInformation: { headers?: WebFetchHeaders }): void;
```

This is optional, I guess?

Reply:
The usages are exclusively Partial but I guess we can make it optional explicitly.